wavefront_django_sdk/middleware.py

```diff
@@ -0,0 +1,353 @@
+                raise AttributeError(
+                    "WF_REPORTER not correctly configured!")
+            elif not isinstance(self.application_tags, ApplicationTags):
+                raise AttributeError(
+                    "APPLICATION_TAGS not correctly configured!")
+            elif not isinstance(self.tracing, DjangoTracing):
+                raise AttributeError(
+                    "OPENTRACING_TRACING not correctly configured!")
+            else:
+                self.APPLICATION = self.application_tags.application or \
+                                   NULL_TAG_VAL
+                self.CLUSTER = self.application_tags.cluster or NULL_TAG_VAL
+                self.SERVICE = self.application_tags.service or NULL_TAG_VAL
+                self.SHARD = self.application_tags.shard or NULL_TAG_VAL
+                self.reporter.prefix = REPORTER_PREFIX
+                self.reg = None
+                if self.is_debug:
+                    self.reg = self.get_conf('DEBUG_REGISTRY')
+                self.reg = self.reg or TaggedRegistry()
+                self.reporter.registry = self.reg
+                if not self.get_conf('WF_DISABLE_REPORTING'):
+                    self.reporter.start()
+                    self.heartbeaterService = HeartbeaterService(
+                        wavefront_client=self.reporter.wavefront_client,
+                        application_tags=self.application_tags,
+                        components=DJANGO_COMPONENT,
+                        source=self.reporter.source)
+                self.tracing._trace_all = getattr(settings,
+                                                  'OPENTRACING_TRACE_ALL',
+                                                  True)
+                initialize_global_tracer(self.tracing)
+                self.MIDDLEWARE_ENABLED = True
+        except AttributeError as e:
+            self.logger.warning(e)
+        finally:
+            if not self.MIDDLEWARE_ENABLED:
+                self.logger.warning("Wavefront Django Middleware not enabled!")
+
+    def __del__(self):
+        if self.reporter:
+            self.reporter.stop()
+        if self.heartbeaterService:
+            self.heartbeaterService.close()
+
+    def process_view(self, request, view_func, view_args, view_kwargs):
+        if not self.MIDDLEWARE_ENABLED:
+            return
+        request.wf_start_timestamp = default_timer()
+        request.wf_cpu_nanos = time.clock()
+
+        entity_name = self.get_entity_name(request)
+        func_name = resolve(request.path_info).func.__name__
+        module_name = resolve(request.path_info).func.__module__
+        self.update_gauge(
+            registry=self.reg,
+            key=self.get_metric_name(entity_name, request) + ".inflight",
+            tags=self.get_tags_map(
+                module_name=module_name,
+                func_name=func_name),
+            val=1
+        )
```

Comment:
Be consistent in closing parens style, either keep it as you do on line 99 or wrap it everywhere 
https://www.python.org/dev/peps/pep-0008/#id16